tests/e2e/scenarios/test_emulate_reborn_provider_contracts.py

```
@@ -0,0 +1,179 @@
+"""Emulate-backed provider contract tests for Reborn-capable integrations.
+
+These tests keep the provider fixture layer honest for integrations that
+Reborn already exposes in the codebase. They assert seeded data and real
+provider-side mutations so the fixtures cannot pass by only booting Emulate.
+"""
+
+import httpx
+
+GOOGLE_TOKEN = "mock-refreshed-access-token"
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to: "GOOGLE_TOKEN"

(S105)